### PR TITLE
Rspec package controller tests and cleanups

### PR DESCRIPTION
**Pull request by bgeuken**
Pair-programmed with @eduardoj

### CODE REVIEW

**Comment by bgeuken** [comment]
@openSUSE/open-build-service Please review. The test suite just fails because of broken backend tests

**Comment by mmohring** [comment]
Hi, I have just reported issue #3042 which seems to be the reason for the backend failure.

**Comment by bgeuken** [comment]
Thanks much

File: src/api/spec/controllers/webui/package_controller_spec.rb [hunk 1]
```diff
@@ -951,20 +951,37 @@ def do_request(params)
...
 
-    it 'lets the user know there was an error' do
-      expect(flash[:error]).to_not be_empty
+    context 'when triggering a rebuild fails' do
+      before do
+        delete :trigger_rebuild, params: { project: source_project, package: source_package }
+      end
+
+      it 'lets the user know there was an error' do
+        expect(flash[:error]).to_not be_empty
```
Comment (bgeuken) [inline src/api/spec/controllers/webui/package_controller_spec.rb]: Welcome in the 'it makes sense to test error messages' camp:-) I am not alone anymore.

I just touched the line when wrapping the code in a context block. But I would be okay to change the error check

File: src/api/spec/controllers/webui/package_controller_spec.rb [hunk 2]
```diff
@@ -159,6 +159,27 @@
     it { expect(response).to redirect_to(package_show_path(project: source_project, package: source_package)) }
   end
 
+  describe "GET #meta" do
+    let(:expected_result) {
```
Comment (bgeuken) [inline src/api/spec/controllers/webui/package_controller_spec.rb]: Nice, didn't thought of that

**Comment by bgeuken** [comment]
@hennevogel Fixed both suggestions now

**Pull request merged**